classes/Provider/Gateways/WebGatewayProvider.php

```diff
@@ -318,10 +314,6 @@ public function boot(Application $app)
             ->bind('reviewer_talks');
         $reviewer->get('/talks/{id}', 'OpenCFP\Http\Controller\Reviewer\TalksController::viewAction')
             ->bind('reviewer_talk_view');
-        $reviewer->post('/talks/{id}/favorite', 'OpenCFP\Http\Controller\Reviewer\TalksController::favoriteAction')
-            ->bind('reviewer_talk_favorite');
-        $reviewer->post('/talks/{id}/comment', 'OpenCFP\Http\Controller\Reviewer\TalksController::commentCreateAction')
-            ->bind('reviewer_talk_comment_create');
```

Comment:
As far as I can tell, both of these routes map to controller actions which exist. Are we sure we want to remove them?

Reply:
They exist on the `Admin\TalksController`, but not on the `Reviewer\TalksController`.

Reply:
Also, I couldn't find any references to the names of both routes apart from the routing configuration itself.

Reply:
This one is my fault 😅 I added routes for the favorite and comment actions for the reviewer when creating all the routes, but later the decision was made to not allow them those privileges.